info/v1/machine.go

```diff
@@ -136,7 +136,7 @@ type MachineInfo struct {
 	CpuFrequency uint64 `json:"cpu_frequency_khz"`
 
 	// The amount of memory (in bytes) in this machine
-	MemoryCapacity int64 `json:"memory_capacity"`
+	MemoryCapacity uint64 `json:"memory_capacity"`
```

Comment:
I know it breaks API, but it can't be done in any other way :( and also int64 is too small for current memory size...